src/applications/appeals/995/utils/focus.js

```diff
@@ -5,15 +5,20 @@ import {
 } from 'platform/utilities/ui';
 import { $ } from 'platform/forms-system/src/js/utilities/ui';
 
+let timer;
```

Comment:
I hate globals, but if they need to exist I think they should have a doc block comment explaining how they are used.

Reply:
Hmmm, I think I had a brain glitch here - in the platform code, the `waitForRenderThenFocus` uses a `setInterval` and it wasn't getting cleared out as expected when the `clearInterval` was called (maybe because it was being called multiple times?). I think this change is because my brain was thinking this was also a `setInterval`... I'll fix it! Thanks for noticing!